src/assets/svg/telegram.svg

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Don't override `telegram.svg` we already have in `master`! An `id` has been added to it manually, which is needed for theming.

Reply:
Fixed